src/ripple/protocol/impl/Quality.cpp

```diff
@@ -64,13 +64,20 @@ Quality::operator--(int)
     return prev;
 }
 
-Amounts
-Quality::ceil_in(Amounts const& amount, STAmount const& limit) const
+template <STAmount (
+    *DivRoundFunc)(STAmount const&, STAmount const&, Issue const&, bool)>
```

Comment:
I usually like passing the callback as a parameter rather than as a template. But I'm also fine with this as-is.